federation/pkg/federation-controller/deployment/deploymentcontroller_test.go

```diff
@@ -174,6 +185,23 @@ func TestDeploymentController(t *testing.T) {
 	deploymentsWatch.Add(dep2)
 	assert.NoError(t, CheckObjectFromChan(cluster1CreateChan, checkDeployment(dep2, 6)))
 	assert.NoError(t, CheckObjectFromChan(cluster2CreateChan, checkDeployment(dep2, 3)))
+
+	// Add new deployment with non-default replica placement preferences.
```

Comment:
Can you fix this comment? It looks like it's currently the same as the code above.